src/PaymentPart/Output/MarkupOutput/AbstractMarkupOutput.php

```diff
@@ -136,45 +191,68 @@ private function hideSeparatorContentIfPrintable(string $paymentPart): string
 
     /**
      * @param Title|Text|Placeholder $element Instance of OutputElementInterface.
+     * @param bool $isReceiptPart
+     * @return string
      */
-    private function getContentElement($element): string
+    private function getContentElement($element, bool $isReceiptPart = false): string
     {
         if ($element instanceof Title) {
-            $elementTemplate = TitleElementTemplate::TEMPLATE;
+            $elementTemplate = $this->getTitleElementTemplate();
+            if(true === $isReceiptPart){
+                $elementTemplate = $this->getTitleElementReceiptTemplate();
+            }
             $elementString = str_replace('{{ title }}', $element->getTitle(), $elementTemplate);
 
             return $elementString;
         }
 
         if ($element instanceof Text) {
-            $elementTemplate = TextElementTemplate::TEMPLATE;
-            $elementString = str_replace('{{ text }}', nl2br($element->getText()), $elementTemplate);
+            $elementTemplate = $this->getTextElementTemplate();
+            $elementTextString = str_replace(array("\r\n", "\r", "\n"), $this->getNewlineElementTemplate(), $element->getText());
+            $elementString = str_replace('{{ text }}', $elementTextString, $elementTemplate);
 
             return $elementString;
         }
 
         if ($element instanceof Placeholder) {
-            $elementTemplate = PlaceholderElementTemplate::TEMPLATE;
+            $elementTemplate = $this->getPlaceholderElementTemplate();
             $elementString = $elementTemplate;
 
-            $svgDoc = new \DOMDocument();
-            $svgDoc->loadXML(file_get_contents($element->getFile()));
-            $svg = $svgDoc->getElementsByTagName('svg');
-            $dataUri = 'data:image/svg+xml;base64,' . base64_encode($svg->item(0)->C14N());
+            $dataUri = 'data:image/png;base64,' .  base64_encode(file_get_contents($element->getFile(Placeholder::FILE_TYPE_PNG)));
+
+            // The svg version works but the images have empty space on top and bottom which makes it unnecessary hard to correctly place them.
+//            $svgDoc = new \DOMDocument();
+//            $svgDoc->loadXML(file_get_contents($element->getFile(Placeholder::FILE_TYPE_SVG))); // Take the png version since the svgs have a bad
+//            $svg = $svgDoc->getElementsByTagName('svg');
+//            $dataUri = 'data:image/svg+xml;base64,' . base64_encode($svg->item(0)->C14N());
```

Comment:
Changing to png again looks like a step back to me. I'd prefer using vector graphics as a default whenever possible. Any suggestion on how to proceed here?

Reply:
I too prefer svg. But since the svg version had these top and bottom spaces the positioning seemed unnecessarily hard to me. Are there any reasons for those spaces that I am missing?
Since I did not want to change your image data i just chose the png version. Without the spaces, the png could easily be switched to svg again in the future.

Reply:
> since the svg version had these top and bottom spaces

I do not know what spaces you are talking about :) Do you have an example?

Reply:
Weird ... there seems to be some kind of odd issue depending in what app you use to open the svg xml.

My IDE phpStorm showed me marks_30x10mm.svg like this:
![image](https://user-images.githubusercontent.com/2971625/90379337-e6bd0280-e07a-11ea-9fcc-008b256f59d6.png)

While marks_30x10mm.png shows as expected:
![image](https://user-images.githubusercontent.com/2971625/90379367-f0df0100-e07a-11ea-9df0-5d0815e14453.png)

When I open the svg file in ImageViewer (Linux Fedora) I get the correct result:
![image](https://user-images.githubusercontent.com/2971625/90379519-1e2baf00-e07b-11ea-8ec3-ff83cbca8359.png)

Also if i use an online svg viewer like https://www.freecodeformat.com/svg-editor.php (with the xml pasted directly into the form field).

Browsers also seem to behave differently when trying to display https://github.com/sprain/php-swiss-qr-bill/blob/master/assets/marks_30x10mm.svg.
FF (on Linux) has the same issue: 
![image](https://user-images.githubusercontent.com/2971625/90379995-cc375900-e07b-11ea-909c-77dde9f76a9d.png)
Vivaldi + Chrome however are correct again:
![image](https://user-images.githubusercontent.com/2971625/90380108-f25cf900-e07b-11ea-8de0-ca81c2642d84.png)

When we figure out a way to get rid of these spaces I see no problem activating the svg version again. Until then I'd use the png version as the more stable solution.

Can you try to reproduce the problem?

Reply:
I got a hunch. Not being an svg expert at all it seems that this hits the old viewPort vs. viewBox discussion (e.g. https://webdesign.tutsplus.com/tutorials/svg-viewport-and-viewbox-for-beginners--cms-30844).

Setting an explicit width and height to the svg and thus changing 
```
<svg version="1.1" id="Ebene_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
	 viewBox="0 0 85.04 28.35" style="enable-background:new 0 0 85.04 28.35;" xml:space="preserve">
```
to
```
<svg version="1.1" width="85.04" height="28.35" id="Ebene_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
	 viewBox="0 0 85.04 28.35" style="enable-background:new 0 0 85.04 28.35;" xml:space="preserve">
<style type="text/css">
```
seems to solve the problem. Not sure about sideeffects yet though. 

Seems that without setting an explicit viewPort (width + height) some programs will set a default size to the SVG. Also the panning and zooming feature of viewBox might have something to do with it.

Reply:
@sprain Would you agree to these changes? I would then commit them as proposed (removing viewBox, setting width + height instead).

Reply:
It seems to work. I prepared the changes and will push them to your fork as soon as I am allowed to do so. Thanks for investigating!

Reply:
I pushed those changes.